src/table/revision.rs

```diff
@@ -9,6 +9,27 @@ use core::fmt;
 #[derive(Copy, Clone, Eq, PartialEq, Ord, PartialOrd)]
 pub struct Revision(u32);
 
+// Allow missing docs, there's nothing useful to document about these
+// constants.
+#[allow(missing_docs)]
+// Allow consistent formatting of the values.
+#[allow(clippy::identity_op)]
+impl Revision {
+    pub const EFI_1_02: Self = Self((1 << 16) | (2));
```

Comment:
Instead of having these `(major << 16) | (minor)` lines which also trigger the `clippy::identity_op` lint, could we instead define a `const fn` for constructing a revision from a major and a minor version?

Reply:
Thanks, good suggestions. I made all the methods of `Revision` `const` and now the `new` method is used for these constants.